e2e-tests/cypress/tests/integration/playbooks/adminconsole/analytics_spec.js

```diff
@@ -9,7 +9,7 @@
 // Stage: @prod
 // Group: @playbooks
 
-describe('admin console', () => {
+describe('admin console', {testIsolation: true}, () => {
```

Comment:
This is the bulk of the changes -- we previously relied on this being on by default, but I didn't want to mess with the stability of the Channels tests right now. If we can slowly start adopting this for new tests, maybe one day it can become the default.